Test/run_tests.sh

```diff
@@ -22,25 +22,21 @@ cd Python_Scripts
 time python script_download_all_sedml_models.py
 time python script_download_all_sbml_biomodels.py
 
-# Run compilation
+# run model regrouping
```

Comment:
@paulstapor 1) The tests do not seem to run through yet. Is that intended? 2) Would make sense to update README.md also already, to keep documentation updated.

Reply:
Agreed. Not 100% perfect yet, as the simulation study is not yet running...

Reply:
Well... Seems like it doesn't help to forget the file extension...